File: jats-demo-model/src/main/java/com/github/pepe79/jats/demo/model/base/VariantValue.java
package com.github.pepe79.jats.demo.model.base;

import java.util.List;

public class VariantValue {

	private List<Component> variants;

	private Object value;

	public Object getValue() {
		return value;
	}

	public void setValue(Object value) {
		this.value = value;
	}

	public List<Component> getVariants() {
		return variants;
	}

	public void setVariants(List<Component> variants) {
		this.variants = variants;
	}

}

File: jats-js/src/test/ts/demo/model/VariantValue.ts
class VariantValue
{
	private value : any;

	private variants : Component[];

	public getValue() { return this.value; }
	public setValue( value : any ) { this.value = value; }

	public getVariants() { return this.variants; }
	public setVariants( variants : Component[] ) { this.variants = variants; }


}
